<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDelivery</class>
 <widget class="QWidget" name="AddDelivery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ДОСТАВКА</string>
  </property>
  <widget class="QPushButton" name="AddDeliveryButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdMaterialLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Матеріалу</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdSuplierLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Постачальника</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdMaterial">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdDelivery">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IdDeliveryLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Поставки</string>
   </property>
  </widget>
  <widget class="QLabel" name="StateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Стан</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdSuplier">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="State">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
